src/test/java/io/vlingo/symbio/store/state/inmemory/InMemoryStateStoreTest.java

```diff
@@ -248,7 +248,7 @@ public void testRedispatch() {
     store.write(entity.id, entity, 1, interest);
 
     try {
-      Thread.sleep(2000);
```

Comment:
Note that this test uses `TestWorld` and thus `TestMailbox`. All messages are delivered directly/synchronously to actors. For the `RedispatchControlActor` test cases the `World.startWithDefaults("test")` should be used.